deploy: history update can fail if container starts slowly, makes superfluous header write

**Describe the bug**
No such container error while executing `inertia <webapp> up`.

Below are the logs around the error.

```
:2019-10-15T03:04:44.288245314Z  ---> db3b423471db
R2019-10-15T03:04:44.288256356Z +Step 8/8 : CMD ["pipenv", "run", "launch"]
E2019-10-15T03:04:44.309796561Z  ---> Running in a3702fc38513
T2019-10-15T03:04:44.376285947Z -Removing intermediate container a3702fc38513
:2019-10-15T03:04:44.376301066Z  ---> 76dafa37d017
G2019-10-15T03:04:44.451185117Z  Successfully built 76dafa37d017
R2019-10-15T03:04:44.461362836Z +Successfully tagged rocket2_rocket2:latest
I2019-10-15T03:04:44.566736851Z container build (387456e0827) has stopped
82019-10-15T03:04:44.687517010Z rocket2 build successful
>2019-10-15T03:04:44.687535627Z Perparing rocket2 container...
92019-10-15T03:04:44.720801699Z rocket2 container created
62019-10-15T03:04:44.720817222Z Starting up rocket2...
�2019-10-15T03:04:45.036853677Z [error 500] failed to update container history following build: (failure fetching container metadata: Error: No such container: )
�2019-10-15T03:04:45.045902766Z 2019/10/15 03:04:45 http: superfluous response.WriteHeader call from github.com/go-chi/render.JSON (responder.go:104)
H2019-10-15T03:04:45.045977059Z [success 201] Project startup initiated!
S2019-10-15T03:04:46.695704835Z container rocket2_nginx_1 (4da9e6c7dac) has stopped
T2019-10-15T03:04:46.695721326Z container stoppage was unexpected, project is active
B2019-10-15T03:04:46.695988524Z Shutting down active containers...
>2019-10-15T03:04:46.701700951Z Stopping /rocket2_rocket2_1...
<2019-10-15T03:04:49.646784528Z Stopping /rocket2_nginx_1...
;2019-10-15T03:04:49.797845222Z Stopping /docker-compose...
S2019-10-15T03:04:50.017728273Z container rocket2_nginx_1 (4da9e6c7dac) has stopped
S2019-10-15T03:04:50.017743715Z container rocket2_nginx_1 (4da9e6c7dac) has stopped
U2019-10-15T03:04:50.017748600Z container rocket2_rocket2_1 (bb784f004e9) has stopped
S2019-10-15T03:04:50.017752936Z container rocket2_nginx_1 (4da9e6c7dac) has stopped
R2019-10-15T03:04:50.017757137Z container docker-compose (11079698074) has stopped
```

**To Reproduce**
Steps to reproduce the behavior:
1. Check out https://github.com/ubclaunchpad/rocket2
2. `inertia init`
3. `inertia remote add aws-ec2` and fill in your informations (`.pem`, username, ip address of AWS EC2 instance)
4. `inertia aws-ec2 init`
5. Set up keys and webhooks on Github's side
6. `inertia aws-ec2 up`
7. See error

**Environment (please complete the following information):**
 - Client OS: `Linux 5.3.1-arch1-1-ARCH`
 - VPS OS: AWS EC2 Amazon Linux AMI 2018.03.0 (HVM), SSD Volume Type - ami-08d489468314a58df
 - Project Details: https://github.com/ubclaunchpad/rocket2
 - Version: latest

Interesting. After having both client and remote on v0.5.3, my `inertia.remotes` aren't read, and so I had to re-init them and re-add them. After entering the information, I got the following error:

```
Port 4303 will be used as the daemon port.
Port 22 will be used as the SSH port.
Run 'inertia remote add' with the -p flag to set a custom Daemon port
of the -ssh flag to set a custom SSH port.
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/ubclaunchpad/inertia/cfg.(*Config).AddRemote(...)
	/home/csys/go/src/github.com/ubclaunchpad/inertia/cfg/config.go:98
github.com/ubclaunchpad/inertia/cmd/inpututil.AddRemoteWalkthrough(0x1046620, 0xc000010010, 0xc0000c86e0, 0x7ffc23c6c4cd, 0x7, 0xe57d38, 0x4, 0xe57655, 0x2, 0xc0000290f8, ...)
	/home/csys/go/src/github.com/ubclaunchpad/inertia/cmd/inpututil/input.go:77 +0xbeb
github.com/ubclaunchpad/inertia/cmd/remote.(*RemoteCmd).attachAddCmd.func1(0xc0001c7180, 0xc00006ff80, 0x1, 0x1)
	/home/csys/go/src/github.com/ubclaunchpad/inertia/cmd/remote/remote.go:91 +0x212
github.com/spf13/cobra.(*Command).execute(0xc0001c7180, 0xc00006ff60, 0x1, 0x1, 0xc0001c7180, 0xc00006ff60)
	/home/csys/go/src/github.com/spf13/cobra/command.go:833 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001c6000, 0x6, 0xc00000f7a0, 0x4072ff)
	/home/csys/go/src/github.com/spf13/cobra/command.go:917 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/home/csys/go/src/github.com/spf13/cobra/command.go:867
main.main()
	/home/csys/go/src/github.com/ubclaunchpad/inertia/main.go:14 +0x4e
```

--- a/daemon/inertiad/daemon/up.go
+++ b/daemon/inertiad/daemon/up.go
@@ -99,9 +99,8 @@ func (s *Server) upHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Update container management history following a successful build and deployment
-	err = s.deployment.UpdateContainerHistory(s.docker)
-	if err != nil {
-		stream.Error(res.ErrInternalServer("failed to update container history following build", err))
+	if err = s.deployment.UpdateContainerHistory(s.docker); err != nil {
+		stream.Println("warning: failed to update container history:", err)
 	}
 
 	stream.Success(res.Msg("Project startup initiated!", http.StatusCreated))

--- a/daemon/inertiad/log/streamer.go
+++ b/daemon/inertiad/log/streamer.go
@@ -68,8 +68,8 @@ func (s *Streamer) GetSocketWriter() (io.Writer, error) {
 }
 
 // Println prints to logger's standard writer
-func (s *Streamer) Println(a interface{}) {
-	fmt.Fprintln(s.Writer, a)
+func (s *Streamer) Println(a ...interface{}) {
+	fmt.Fprintln(s.Writer, a...)
 }
 
 // Error directs message and status to http.Error when appropriate

